prettier.js

```diff
@@ -1,11 +1,14 @@
 module.exports = {
+  plugins: [
+    'stylelint-prettier',
+  ],
   extends: [
     './index',
     'stylelint-config-prettier',
   ],
-
-  // conflicts with prettier formatting
   rules: {
+    'prettier/prettier': true,
```

Comment:
What default prettier options does this get set to? If a consuming project has their own respective `.prettierrc` will those options be used by `stylelint-prettier`

Reply:
This sets no prettier options by default. It will pick up options used by .prettierrc.